beacon-chain/blockchain/execution_engine.go

```diff
@@ -324,8 +324,8 @@ func (s *Service) getPayloadAttribute(ctx context.Context, st state.BeaconState,
 
 	var attr payloadattribute.Attributer
 	switch st.Version() {
-	case version.Deneb:
-		withdrawals, err := st.ExpectedWithdrawals()
+	case version.Deneb, version.Electra:
```

Comment:
Why do we want to add `version.Electra` here? I feel like this PR should be isolated to only `Electra: BeaconState`. If we do want to add it, I'd suggest we: 
- Add a unit test to `getPayloadAttribute`
- Also add `version.Electra` to L345 below

Reply:
Reverted this and the other one in 2d6b28c6497cdd4e0781e41760bf45b625743e45